libinteract/acPSN.py

```diff
@@ -78,19 +78,19 @@ def __init__(self):
     ######################### PRIVATE METHODS #########################
 
 
-    def _set_i_min(self,
-                   i_min):
+    def _check_i_min(self,
+                     i_min):
         """Check and return a single i_min value.
         """
         
         # If the i_min is neither an integer, a float or None, raise
         # an error
-        if not isinstance(i_min, (int, float, type(None))):
-            errmsg = "A single i_min must be int, float or None."
-            raise TypeError(errmsg)
+        if not isinstance(i_min, (int, float)):
+            errstr = "A single i_min must be int, float or None."
+            raise TypeError(errstr)
         
-        # Set the i_min to -inf if it was None
-        i_min = float(i_min) if i_min else -np.inf
+        # Set the i_min to 0.0 if it was None
+        i_min = float(i_min) if i_min is not None else 0.0
```

Comment:
I don't think `i_min` could ever be `None` since you're checking if it's a number already

Reply:
It could, but I forgot to include it as a possibility in the previous check. Therefore, I included the possibility of the i_min being `None` in the previous check and separated line 93 in a if...else block where not only the i_min is set to 0.0 if `None` was passed (or the i_min gets converted to a float if an int or a float was passed) but it is written in the log which case was encountered.